geode-connectors/src/distributedTest/java/org/apache/geode/connectors/jdbc/internal/cli/DestroyMappingCommandDunitTest.java

```diff
@@ -231,22 +332,46 @@ private static void validateRegionAlteredInClusterConfig(boolean synchronous) {
     }
   }
 
-  private void verifyQueueRemoved(InternalCache cache) {
-    String queueName = CreateMappingCommand.createAsyncEventQueueName(TEST_REGION);
-    assertThat(cache.getAsyncEventQueue(queueName)).isNull();
+  private void verifyQueueRemoved(InternalCache cache, String regionName, boolean exists) {
+    if (regionName == null) {
```

Comment:
Instead of having the null check here, can we just pass `TEST_REGION` as parameter wherever appropriate? e.g. `verifyQueueRemoved(cache, null, false);` becomes `verifyQueueRemoved(cache, TEST_REGION, false);`. This makes the code easier to read and understand.

Reply:
✅